Makefile

```diff
@@ -2,6 +2,10 @@ VERSION := $(shell cat VERSION)
 PREFIX ?= /usr/local
 BINDIR ?= ${PREFIX}/bin
 LIBEXECDIR ?= ${PREFIX}/libexec
+GO ?= go
+PROJECT := github.com/containers/conmon
+
```

Comment:
I would add here:
```suggestion
BUILDTAGS := containers_image_ostree_stub
```

Reply:
why that tag in particular?

Reply:
Because it does not build on my machine without the dependency installed :innocent:

Reply:
this patch or cri-o? if cri-o, I meant to not have any cri-o imports in this repo (first version of this PR was a bit sloppy :\ ), and the code here should not have containers/image associated with it

Reply:
Ah alright, so then we do not need it. Yes I guess it's because of CRI-O.